source/frontend/src/features/documents/ComposedDocument.ts

```diff
@@ -235,6 +237,7 @@ export class DocumentUpdateFormData {
     return {
       documentId: this.documentId,
       mayanDocumentId: this.mayanDocumentId,
+      documentTypeId: stringToNumber(this.documentTypeId),
```

Comment:
if empty i think this will set the document typeid to zero. If that happens, will the update be successful on the backend?

Reply:
true, it does fail. I've added validations to mark as required.